src/app/pages/account/signup/page.jsx

```diff
@@ -1,9 +1,248 @@
+          <div className={styles.inputBox}>
+          <AiOutlineMail className={styles.icon}/>
+            <input type="email" onChange={handleEmailChange} value={emailInput} maxLength={30} placeholder='Email' className={styles.input} required/>
+            <p className={styles.validation}>{validationEmail ? valiEmailMessage : ''}</p>
+          </div>
+          <div className={styles.inputBox}>
+          <PiIdentificationCardLight className={styles.icon}/>
+            <input type="text" onChange={handleCiChange} value={ciInput} placeholder='CI' className={styles.input} maxLength={16} inputMode='numeric' required/>
+            <p className={styles.validation}>{validationCi ? validCiMessage : ''}</p>
+          </div>
+          <div className={styles.inputBox}>
+            <AiOutlineLock className={styles.icon} />
+            <input
+              type={passwordVisible ? 'text' : 'password'}
+              placeholder="Password"
+              className={styles.input}
+              value={passwordInput}
+              onChange={handlePasswordChange}
+              maxLength={16}
+              required
+            />
+            <p className={styles.validation}>
+              {validationPassword ? valiPasswordMessage : ''}
+            </p>
+            {passwordVisible ? (
+              <AiOutlineEyeInvisible
+                className={styles.iconLock}
+                onClick={handlePasswordToggle}
+              />
+            ) : (
+              <AiOutlineEye className={styles.iconLock} onClick={handlePasswordToggle} />
+            )}
+          </div>
+          <div className={styles.genderBox}>
+        <label className={styles.label}>Gender:</label> 
+        
+        <div className={styles.radioGroup} required>
+          <label>
+            <input
+              type="radio"
+              value="Male"
+              checked={selectedGender === "Male"}
+              onChange={handleGenderChange}
+
+            />
+             Male
+          </label>
+          <label>
+            <input
+              type="radio"
+              value="Female"
+              checked={selectedGender === "Female"}
+              onChange={handleGenderChange}
+            />
+            Female
+          </label>
+          <label>
+            <input
+              type="radio"
+              value="RatherNotSay"
+              checked={selectedGender === "RatherNotSay"}
+              onChange={handleGenderChange}
+            />
+            Rather not say
+          </label>
+        </div>
+      </div>
+          <button className={styles.button}>Create account</button>
+        </form>
+        <div className={styles.bottomSignUp}>
+          <p className={styles.signUpTxt}>Already have an account?</p>
+          <Link href={"/pages/account/login"} className={styles.signUpButton}>Log In</Link>
+      </div>
+      </div>
+    </div>
   )
 }
 
-export default SingUp
\ No newline at end of file
+export default SignUp
```

Comment:
It would be good if you do not call it below but above for example: export default default function Example(){code} and this line of code delete it